app/design/frontend/base/default/template/oyst/oneclick/cart/oneclick.phtml

```diff
@@ -14,7 +14,11 @@
  */
 ?>
 
-<div id="oyst-1click-button-wrapper">
+<?php
+echo $this->addButtonWrapperStyles('checkout_cart_button_class', 'checkout_cart_button_styles');
+?>
+
+<div id="oyst-1click-button-wrapper" <?php echo $this->getWrapperClass(); ?>>
```

Comment:
change for `<div id="oyst-1click-button-wrapper" <?php echo $this->getButtonCustomization(); ?>>`
to be compatible with remove this method, duplicate with https://github.com/oystparis/oyst-1click-magento/pull/252/files